tests/src/services/index.spec.ts

```diff
@@ -0,0 +1,14 @@
+import mock = require("supertest");
+
+import { app } from "../../../scripts/bin/app";
+
+describe("general api checks", async () => {
+	test("unknown api endpoints should return 404", done => {
+		mock(app)
+			.get("/v1/no_such_page")
+			.expect(404)
+			.then(() => {
+				done();
```

Comment:
I'm pretty sure jest lets you skip the `done` function: https://facebook.github.io/jest/docs/en/asynchronous.html#async-await